template.yml

```diff
@@ -40,62 +49,65 @@ Resources:
       CodeUri: ./backend-lambda
       Environment:
         Variables:
-          DATABASE_URL: !Ref DATABASE_URL
+          DATABASE_URL: !Ref DatabaseUrl
       Events:
         Api:
           Type: Api
           Properties:
-            Path: /myendpoint
-            Method: get
+            Path: /
+            Method: ANY
 
   RestApi:
     Type: AWS::Serverless::Api
     Properties:
       StageName: prod
       Auth:
-        DefaultAuthorizer: IPAuthorizerLambdaFunction
+        DefaultAuthorizer: IpAuthorizerLambdaFunction
       DefinitionBody:
         swagger: "2.0"
         info:
           title: "RestApi"
         paths:
-          /myendpoint:
-            get:
+          /products:
+            get: &getProducts
               responses:
                 "200":
-                  description: "Successful response for GET request"
+                  description: "Successful response for GET products request"
               x-amazon-apigateway-integration:
-                uri: !Sub "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${BackendLambdaFunctionArn}/invocations"
+                uri: !Sub "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${BackendLambda.Arn}/invocations"
                 httpMethod: "GET"
                 type: "aws_proxy"
-            post:
+            post: &postProducts
               responses:
                 "200":
-                  description: "Successful response for POST request"
+                  description: "Successful response for POST products request"
               x-amazon-apigateway-integration:
-                uri: !Sub "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${BackendLambdaFunctionArn}/invocations"
+                uri: !Sub "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${BackendLambda.Arn}/invocations"
                 httpMethod: "POST"
                 type: "aws_proxy"
-        responses:
-          "4XX":
-            description: Default response for 4xx errors
-          "5XX":
-            description: Default response for 5xx errors
-        x-amazon-apigateway-gateway-responses:
-          DEFAULT_4XX:
-            responseTemplates:
-              application/json: '{"message": "Client Error: {statusCode}"}'
-          DEFAULT_5XX:
-            responseTemplates:
-              application/json: '{"message": "Server Error: {statusCode}"}'
+          /product:
+            post: *postProducts
```

Comment:
Update these methods as commented on in the backend lambda index.js.

Reply:
done